.github/workflows/explorer-dev.yaml

```diff
@@ -0,0 +1,21 @@
+name: Enable Explorer maintenance on DEV
```

Comment:
Engineer should be able to select in the Dropdown:
1. App
2. Env

Reply:
@AndreiNavoichyk, I suggest to run a minimal working version to see how it works and then see how we can optimize it.